src/containers/TransactionPage/TransactionPage.duck.js

```diff
@@ -470,7 +483,7 @@ export const loadData = params => dispatch => {
   const txId = new UUID(params.id);
 
   // Clear the send error since the message form is emptied as well.
-  dispatch(setInitialValues({ sendMessageError: null, sendReviewError: null }));
+  dispatch(clearErrors());
```

Comment:
👍 
Did you check if any other files had setIntialValues called inside loadData?

Reply:
@Gnito good question. This seems to be the only loadData implementation that does this.